### ISSUE TITLE
Add Frameworks/Tools metadata field

### ISSUE THREAD

**Issue by palash25** [coala/coala-bears#2210]
I was looking at the GitCommitBear and noticed that in the bear metadata mentioned git as a language.
But git cannot be classified as a programming language.
IMO this `LANGUAGES = {'Git'}` should look something like this `TOOLS = {'Git'}`.

**Comment by Makman2** [comment]
Good point actually... but we don't handle a `TOOLS` field yet from coala side ;)

I'm actually not sure what we should do here...

CC @jayvdb

**Comment by palash25** [comment]
> but we don't handle a TOOLS field yet from coala side ;)
btw can you tell me where do all the other metadata fields reside?

**Comment by Makman2** [comment]
Those fields are defined in the bear, however they have to properly integrate into coala's features (like filtering for bears, displaying a summary for a bear, etc). Especially filtering is important, as people need a way to find the bear if they want to check git-related things.
